src/plugin-slots/AdditionalCourseContentPluginSlot/README.md

```diff
@@ -4,3 +4,73 @@
 
 ### Slot ID Aliases
 * `additional_course_content_plugin`
+  
+## Description
+
+This slot is used to add a custom block in the **Content permissions** section of the **page & resources** page.
```

Comment:
Does anyone know why we even have "Content permissions" as a separate section? It doesn't seem that useful to me to have one or two cards separate from the others. Why are the permissions settings not just managed within the "regular" settings for the particular app?